src/handlers/deeplinks.ts

```diff
@@ -94,7 +94,7 @@ export default async function handleDeeplink(
         if (addressOrENS) {
           const isValid = await checkIsValidAddressOrDomain(addressOrENS);
           if (isValid) {
-            return Navigation.handleAction(Routes.SHOWCASE_SHEET, {
+            return Navigation.handleAction(Routes.PROFILE_SHEET, {
```

Comment:
could you add a check for the profile feature flag? if it's off we should use the old route